tests/cypress/latest/e2e/unit_tests/upgrade-operator.spec.ts

```diff
@@ -52,14 +52,14 @@ describe('Elemental operator upgrade tests', () => {
           cy.clickButton('Upgrade');
           cy.contains('.header > .title', 'elemental-operator');
           cy.clickButton('Next');
-          cy.get('[data-testid="string-input-channel.repository"]')
+          cy.get('[data-testid="string-input-channel.image"]')
             .clear()
-          cy.get('[data-testid="string-input-channel.repository"]')
-            .type('rancher/elemental-channel')
+          cy.get('[data-testid="string-input-channel.image"]')
+            .type('registry.opensuse.org/isv/rancher/elemental/dev/containers/rancher/elemental-channel')
```

Comment:
So not possible to test upgrade from `Stable` to `Staging` with this hardcoded? Maybe something like [here](https://github.com/rancher/elemental/blob/9a5f23fc12822fb581f305ad64d79459c02c0bbc/tests/cypress/latest/support/commands.ts#L448) could be possible?

Reply:
Yes, I was aware of this one, I will open an issue to improve it, thx.